system-addon/lib/TopSitesFeed.jsm

```diff
@@ -137,8 +136,12 @@ this.TopSitesFeed = class TopSitesFeed {
       data: this._getPinnedWithData()
     }));
   }
-  onAction(action) {
+  async onAction(action) {
     switch (action.type) {
+      case at.INIT:
```

Comment:
I used Screenflow, but any screen recording software with editing should be good.